### PR TITLE
various grammar and typo fixes.

### PR DESCRIPTION
**Pull request by xPolar**
Compilation of various other PRs and issues as well as items from the "Found a typo?" discussion.

### CODE REVIEW

**Review by Brendonovich — APPROVED** [review]

File: apps/landing/src/pages/index.tsx [hunk 1]
```diff
@@ -98,9 +98,9 @@ function Page() {
 				</Button>
 			</div>
 			<p className="px-6 mt-3 text-sm text-center opacity-75 text-gray-450">
-				Coming soon on macOS, Windows and Linux.
+				Coming soon to macOS, Windows, and Linux.
 				<br />
-				Shortly after to iOS & Android.
+				Then shortly after to iOS & Android.
```
Comment (maxichrome) [inline apps/landing/src/pages/index.tsx:103]: This actually makes _less_ grammatical sense to me than the previous version.

```suggestion
				Shortly after to iOS & Android.
```

If that original wording really was unclear, you could write this instead:

```suggestion
				Shortly thereafter to iOS & Android.
```

File: docs/architecture/virtual-filesystem.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ struct File {
 
 > ~~It is impossible to have a unique constraint at a database level for the `partial_checksum` however we can asynchronously resolve conflicts by querying for duplicates and generating full checksums at a later date.~~
 >
-> For synchronization of this resource we can tolerate temporary duplicates, any client can calculate that two files resources are duplicate and merge them into a single resource. In turn, triggering a shared data merge operation, whereby the older record is prioritised at a property level during the merge.
+> For synchronization of this resource we can tolerate temporary duplicates, any client can calculate that two files resources are duplicate and merge them into a single resource. In turn, triggering a shared data merge operation, whereby the older record is prioritized at a property level during the merge.
```
Comment (maxichrome) [inline docs/architecture/virtual-filesystem.md:41]: do we want to use American, Canadian, or British English conventions in documentation? we should have some general standard for spell checking purposes and all

File: apps/landing/src/pages/index.tsx [hunk 3]
```diff
@@ -98,9 +98,9 @@ function Page() {
 				</Button>
 			</div>
 			<p className="px-6 mt-3 text-sm text-center opacity-75 text-gray-450">
-				Coming soon on macOS, Windows and Linux.
+				Coming soon to macOS, Windows, and Linux.
```
Comment (maxichrome) [inline apps/landing/src/pages/index.tsx:101]: Oxford comma W

**Review by maxichrome — CHANGES_REQUESTED** [review]
All is acceptable to me bar the "shortly after" comment.

File: docs/architecture/virtual-filesystem.md (hunk 2 above)
Comment (maxichrome) [reply docs/architecture/virtual-filesystem.md:41]: I'll make a task for this and we can discuss sometime in the coming days

Comment (maxichrome) [reply docs/architecture/virtual-filesystem.md:41]: Created as ENG-79

**Review by maxichrome — APPROVED** [review]
All previous comments were resolved!

**Pull request merged**